adoc/admin-updates.adoc

```diff
@@ -1,12 +1,19 @@
-== Control Plane
+= Handling updates
```

Comment:
if you use this `=` header it causes to show an empty page, you have to treat `==` as the largest header here, and use smaller headers from there on. then it renders correctly again

![cluster-updates-docs](https://user-images.githubusercontent.com/5364817/60887976-ff42d280-a255-11e9-9091-b2773eccaa57.png)

Reply:
Fixed